source/index.txt

```diff
@@ -20,8 +16,76 @@ MongoDB in Ruby. To work with {+odm+} from the command line using
    Secure Your Data </security>
    Integrations & Tools </integrations-tools>
    API Documentation </api>
-   What's New </whats-new>
    Compatibility </compatibility>
+   What's New </whats-new>
    Issues & Help </issues-and-help>
-   /additional-resources
-   /ecosystem
+   View the Source <https://github.com/mongodb/mongoid>
+
+Introduction
+------------
+
+Welcome to the documentation site for {+odm+}. {+odm+} is the officially
+supported object-document mapper (ODM) for MongoDB in {+language+}.
+
+You can add {+odm+} to your application to connect your {+language+} to
```

Comment:
i: i can't tell what the end of this line should be; maybe '{+language+} application'? or just 'to connect it'?

Reply:
oops yes should be Ruby app